context/app/static/js/components/home/HuBMAPDatasetsChart/HuBMAPDatasetsChart.tsx

```diff
@@ -0,0 +1,202 @@
+import React, { ComponentProps } from 'react';
+import Paper from '@mui/material/Paper';
+
+import ChartWrapper from 'js/shared-styles/charts/ChartWrapper';
+import { useSelectedDropdownIndex } from 'js/shared-styles/dropdowns/DropdownListbox';
+import HorizontalStackedBarChart from 'js/shared-styles/charts/HorizontalStackedBarChart';
+import { useChartPalette } from 'js/shared-styles/charts/HorizontalStackedBarChart/hooks';
+import { ChartArea } from 'js/shared-styles/charts/HorizontalStackedBarChart/style';
+import { useBandScale, useLinearScale, useOrdinalScale } from 'js/shared-styles/charts/hooks';
+import Skeleton from '@mui/material/Skeleton';
+import { TooltipData } from 'js/shared-styles/charts/types';
+import Typography from '@mui/material/Typography';
+import { AnyD3Scale } from '@visx/scale';
+import { getSearchURL } from 'js/components/organ/utils';
+import HuBMAPDatasetsChartDropdown from '../HuBMAPDatasetsChartDropdown';
+import {
+  assayTypeQuery,
+  donorSexQuery,
+  analyteClassQuery,
+  processingStatusQuery,
+  AssaysQueryKey,
+  DonorSexQueryKey,
+  AnalyteClassQueryKey,
+  ProcessingStatusQueryKey,
+  donorRaceQuery,
+  DonorRaceQueryKey,
+} from './queries';
+import {
+  AggregatedDatum,
+  getKeysFromAggregatedData,
+  useAggregatedChartData,
+  useDatasetTypeMap,
+  useOrganOrder,
+  useSearchDataRange,
+} from './hooks';
+
+const margin = { top: 40, right: 50, bottom: 64, left: 100 };
```

Comment:
Perhaps comment why these values were chosen?

Reply:
Good call; most of these initial values were inherited from the original implementation. I've adjusted the values for all directions except the bottom to match Figma (32px) and added an explanation for why the bottom margin is greater than the others.

![image](https://github.com/hubmapconsortium/portal-ui/assets/19957804/d4d8208f-6ffb-40b0-88bb-93f811ab4625)